File: chat/src/main/java/com/woobeee/chat/dto/PostExportDto.java
package com.woobeee.chat.dto;

import com.fasterxml.jackson.annotation.JsonInclude;

import java.time.LocalDateTime;

@JsonInclude(JsonInclude.Include.NON_NULL)
public record PostExportDto(
        Long id,
        String url,
        String titleKo,
        String titleEn,
        String textKo,
        String textEn,
        Long views,
        LocalDateTime createdAt,
        LocalDateTime updatedAt,
        String categoryNameKo,
        String categoryNameEn
) {}

File: chat-v2/src/main/kotlin/com/woobeee/chatv2/dto/PostExportDto.kt
package com.woobeee.chatv2.dto

import com.fasterxml.jackson.annotation.JsonInclude
import java.time.LocalDateTime

@JsonInclude(JsonInclude.Include.NON_NULL)
data class PostExportDto(
    val id: Long?,
    val url: String?,
    val titleKo: String?,
    val titleEn: String?,
    val textKo: String?,
    val textEn: String?,
    val views: Long?,
    val createdAt: LocalDateTime?,
    val updatedAt: LocalDateTime?,
    val categoryNameKo: String?,
    val categoryNameEn: String?
)
